styles/globals.css

```diff
@@ -0,0 +1,71 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+
+@font-face {
+  font-family: "Neufile";
+  src: url('../public/assets/font/NeufileGrotesk-Regular.eot');
```

Comment:
you can directry import like this `/assets/font/NeufileGrotesk-Regular.eot` directry without specifing public directory becouse it is publicly accessible